tests/core/array/test_reshape.py

```diff
@@ -99,6 +99,8 @@ def _strip_ones(shape, block_shape):
                 new_shape = new_shape[:pos] + ones + new_shape[pos:]
                 new_block_shape = list(block_shape)
                 new_block_shape = new_block_shape[:pos] + ones + new_block_shape[pos:]
+                print("Shapes: {} {}".format(shape, new_shape))
+                print("Block Shapes: {} {}".format(block_shape, new_block_shape))
                 new_arr = arr.reshape(new_shape, block_shape=new_block_shape)
```

Comment:
Have you confirmed that all of these inputs trigger a simple reshape? If they don't, this test may take a very long time.

Reply:
Done.